kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/enums/NM8021xEAP.java

```diff
@@ -0,0 +1,11 @@
+package org.eclipse.kura.nm.enums;
+
+public enum NM8021xEAP {
+    leap, 
+    md5,
+    tls, 
+    peap,
+    ttls,
+    pwd,
+    fast
```

Comment:
Please use all caps enums to be consistent with the already established convention and to make Sonar happy.

See https://github.com/eclipse/kura/blob/develop/kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/enums/NMSettingIP6ConfigPrivacy.java and https://github.com/eclipse/kura/blob/develop/kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/enums/NMSettingIP6ConfigAddrGenMode.java for reference